includes/env_access.h

```diff
@@ -0,0 +1,10 @@
+#ifndef ENV_ACCESS_H
+# define ENV_ACCESS_H
+
+#include "env.h"
+
+char	*env_get_user(t_env *_env);
+char	*env_get_host(t_env *_env);
+char	*env_get_working_dir(t_env *_env);
```

Comment:
wouldn't env_get_current_dir make more sense? working directory could mean a bunch of things